### PR TITLE
Add Field names from Schema to FileMan summary page

### PR DESCRIPTION
**Pull request by josephsnyder**
Add the field names in the summary page for each FileMan file.
Make each page a "large file" to bring in the data at page load time to
eliminate some page loading time.

### CODE REVIEW

File: Utilities/Dox/PythonScripts/FileManDataToHtml.py [hunk 1]
```diff
@@ -714,52 +832,48 @@ def findSubValue(self,dataEntry,search,multval, id):
   def _generateDataTableHtml(self, fileManData, fileNo, outDir):
     isLargeFile = len(fileManData.dataEntries) > 4500
     tName = normalizePackageName(fileManData.name)
+    print "Writing info for %s" % fileNo
```
Comment (BetsyMcPhail) [inline Utilities/Dox/PythonScripts/FileManDataToHtml.py]: Should this be logged instead of printed?

Comment (josephsnyder) [reply Utilities/Dox/PythonScripts/FileManDataToHtml.py]: Most likely.  I'll remove it.

**Comment by BetsyMcPhail** [comment]
Like we discussed, the first column in ..../files/19_081/19.081.html is incorrect

**Comment by josephsnyder** [comment]
Thanks for the review @BetsyMcPhail, Please take a look again when you have time

**Pull request merged**